hms-api/src/handler/authorize.ts

```diff
@@ -0,0 +1,59 @@
+import {verify} from 'jsonwebtoken';
```

Comment:
so likely how we should do it is that we have an authorize function that checks the configuration and then decides which concrete authorizer to choose.

so then we would have this class, but then also a activeDirectoryAuthorizer.  We could have another one that is a allPassAuthorizer (that just always passes, we can use it for testing purposes maybe :thinking:)

Reply:
in the configuration there would be something like an `AUTHORIZER_TYPE` which could have `ACTIVE_DIRECTORY` and `ALWAYS_PASS` as possible values